### ISSUE TITLE
Investigate npm deprecations

### ISSUE THREAD

**Issue by dignifiedquire** [ipfs/js-ipfs#632]
Running a fresh install pops up a lot of npm deprecations, we should investigate where they come from and file issues with the relevant packages if needed.

```
npm WARN deprecated lodash.contains@2.4.3: This package is discontinued. Use lodash.includes@^4.0.0.
npm WARN deprecated node-uuid@1.4.7: use uuid module instead
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```

**Comment by quaelin** [comment]
The warning about `lodash.contains` is coming from `js-libp2p-websockets`, and I've submitted a fix here: libp2p/js-libp2p-websockets#39

I think the rest are coming from `gulp` due to its dependency on the very old version of `vinyl-fs`.  There's a related gulp ticket to upgrade that here: gulpjs/gulp#1604

**Comment by diasdavid** [comment]
awesome @quaelin :D
